codespell_lib/data/dictionary.txt

```diff
@@ -9602,6 +9602,12 @@ distingusih->distinguish
 distinquish->distinguish
 distinquishable->distinguishable
 distintions->distinctions
+distirbute->distribute
+distirbuted->distributed
```

Comment:
I guess "disturbed" is probably too-different to suggest here?

Reply:
I agree

Reply:
Yes, it is.